docs/Installation/Installation.md

```diff
@@ -280,7 +287,10 @@ Follow these steps:
 3.  Return to `SPECTRE_ROOT`, and create a build dir by running
     `mkdir build && cd build`
 4.  Build SpECTRE with
-    `cmake -D CHARM_ROOT=CHARM_DIR/ARCH_OPTS SPECTRE_ROOT`
+```
+    cmake -D CMAKE_Fortran_COMPILER=gfortran-8 \
```

Comment:
I don't think this should be changed here since these instructions are not specific to docker or singularity